communities/microgalaxy/lab/usegalaxy.eu/templates/footer.html

```diff
@@ -42,7 +42,7 @@
     <p>
       <span>
         <i class="fab fa-github"></i>
-        <a href="https://github.com/galaxyproject/galaxy_codex/tree/subdomain_unification/subdomains/singlecell">Edit this page on GitHub</a>
+        <a href="https://github.com/galaxyproject/galaxy_codex/tree/subdomain_unification/subdomains/microgalaxy">Edit this page on GitHub</a>
```

Comment:
Have you tried this link? The branch should be `main` and the path is wrong `communities/microgalaxy/lab`

Reply:
yeah this was changed